gulpfile.js

```diff
@@ -187,6 +187,10 @@ gulp.task('watch', function() {
   gulp.watch(paths.src, ['modules']);
 });
 
+gulp.task('dev', function() {
```

Comment:
Having a task that also do dist is very important for speed up development specially when doing tdd or exploratory testing using one of the example html files :)